src/test/java/edu/byu/cs/server/exception/ResponseParseException.java

```diff
@@ -0,0 +1,7 @@
+package edu.byu.cs.server.exception;
+
+public class ResponseParseException extends Exception {
+    public ResponseParseException(String message, Exception cause) {
+        super(message, cause);
+    }
+}
```

Comment:
Do these need to be separate Exceptions? For the chess passoff jar, I did that so students could have a slightly better idea what's going on. Since this should only matter to TA's (and it's only be used by tests), would there be any problems with letting the TestServerFacade throw IOExceptions rather than custom test exceptions?

Reply:
I guess not? I'm not sure I would normally categorize server communication as being I/O, but I suppose it meets the definition, and it's probably better than one-time-use classes. I do want to catch and rethrow though for the sake of error message clarity

Reply:
Hmm. That's not exactly what I meant but this seems usable. If you want to stick with the catch and rethrow then one test-only exception might be warranted (if you want to, this looks fine too)